docs/examples/orms.md

```diff
@@ -0,0 +1,55 @@
+!!! warning "🚧 Work in Progress"
+    This page is a work in progress. More examples will be added soon.
+
+Pydantic serves as a great tool for defining models for ORM (object relational mapping) libraries.
+ORMs are used to map objects to database tables, and vice versa.
+
+## SQLAlchemy
+
+Pydantic pairs quite well with SQLAlchemy, as it can be used to define the schema of the database models.
+
+Here's a simple example of how you can use a Pydantic model to validate data from a SQLAlchemy model.
+In this example, we take advantage of field aliases to name a `Column` after a reserved SQLAlchemy field, thus avoiding conflicts.
```

Comment:
I think most people would consider the following code fairly clunky — you've basically just copy/pasted the same schema into both a pydantic model and a sqlalchemy model. In particular, I think most people would take issue with the phrasing "it can be used to define the schema of the database models" — it's not being used for that, it just happens to be duplicating the schema, and once you've done that, you can use it for validation before converting the data into a sqlalchemy model type.

I'm not saying we shouldn't include an example here, but if it were up to me I'd phrase it more as:
> It is _possible_ to use pydantic and sqlalchemy together, and if you do, you'll need to do one of two things:
> * Define a pydantic model and a sqlalchemy model separately and manually keep them in sync
> * Use tiangolo's `SQLModel` library

In general I personally find the use of Pydantic and Sqlalchemy together pretty unfortunate — you either have to define everything twice, or use SQLModel, and in my experience SQLModel has a lot of little nuisances (not least of which being — most of the APIs are poorly handled by pycharm. Not sure if it works better with other type checkers, but I feel like I've seen a lot of type: ignores in code that uses it.)

That's not to say we shouldn't include examples, but I personally wouldn't want to try to sell it as some excellent integration, just say how it works and let people decide for themselves if it's good or not.

Reply:
This makes sense to me. We basically copy / pasted this example from a different page where we took it out because it wasn't super relevant.

Maybe for now, we should just leave a ref to `SQLModel`, and expand on this later if there's a more compelling case?

Reply:
I don't think that's a bad idea. But I also think it's okay to include an example showing that this is what you need to do, so that people who are writing code aren't like "surely there's a better way to do this?!?!", as their reaction might be. I would just frame the example from the perspective of — "Here's how we recommend using Pydantic models alongside SQLAlchemy. If you find this duplication of structs between Pydantic and SQLAlchemy frustrating, you may want to look into `sqlmodel`, which is designed to let you use a single type as both a Pydantic model and a SQLAlchemy model.

Reply:
Awesome. I've added a warning + gave some more context re duplication.